packages/daimo-api/src/contract/requestIndexer.ts

```diff
@@ -203,6 +223,52 @@ export class RequestIndexer {
     return statuses as DaimoRequestV2Status[];
   }
 
+  private storeReqByAddress(address: Address, id: bigint) {
+    const existingSet = this.requestsByAddress.get(address);
+
+    if (!existingSet) {
+      this.requestsByAddress.set(address, new Set([id]));
+    } else {
+      this.requestsByAddress.set(address, existingSet.add(id));
```

Comment:
this line can just be `existingSet.add(id)`

that object is already in `requestsByAddress`

Reply:
True. Nice catch.